v-next/hardhat/test/fixture-projects/cli/parsing/subtask-help/hardhat.config.ts

```diff
@@ -0,0 +1,10 @@
+import {
+  HardhatUserConfig,
+  emptyTask,
+} from "@nomicfoundation/hardhat-core/config";
+
+const customTask = emptyTask("empty-task", "empty task description").build();
```

Comment:
@alcuadrado  this is how I build the empty task.
The code logic, I think, should go in this if branch:

![image](https://github.com/NomicFoundation/hardhat/assets/18092467/75682f47-b3ef-491a-9d10-729f9c32ec5d)

But it does not, [this test is failing](https://github.com/NomicFoundation/hardhat/pull/5380/files#diff-0c92acf051033f1d9388e08f2cf7363ce89dc689512eb4c57ae00847c1e4bd20:~:text=it(%22should%20print%20an%20help%20message%20for%20the%20task%27s%20subtask%22%2C%20async%20function%20()%20%7B). Can you double check if I'm misinterpreting how empty task should work or if I should open an issue?